### PR TITLE
Feast or Famine is shit. Makes CE hardsuit raditation proof hat wise, buffes its rest of body by 5

### LINKED ISSUE #10129: CE hardsuit not protecting from rads

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
more at 11

## About The Pull Request
closes #10129 
Makes the hat able to be rad proof
Makes the body have 5 more rad protection too

## Why It's Good For The Game

Lets not kill people with Feast or Famine shit they CANT deal with k thx

## Changelog
:cl:
tweak: CE hardsuit is now more rad-proof
/:cl:

### CODE REVIEW

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 1]
```diff
@@ -208,11 +208,11 @@
 	//Chief Engineer's hardsuit
 /obj/item/clothing/head/helmet/space/hardsuit/engine/elite
 	name = "advanced hardsuit helmet"
-	desc = "An advanced helmet designed for work in a hazardous, low pressure environment. Shines with a high polish."
+	desc = "An advanced helmet designed for work in a hazardous, low pressure environment. Shines with a high polish, and an added layer of radiation shielding."
```
Comment (Ghommie) [inline code/modules/clothing/spacesuits/hardsuit.dm]: no need to modify the desc.

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 2]
```diff
@@ -221,7 +221,7 @@
 	name = "advanced hardsuit"
 	desc = "An advanced suit that protects against hazardous, low pressure environments. Shines with a high polish."
 	item_state = "ce_hardsuit"
-	armor = list("melee" = 40, "bullet" = 5, "laser" = 10, "energy" = 5, "bomb" = 50, "bio" = 100, "rad" = 90, "fire" = 100, "acid" = 90)
+	armor = list("melee" = 40, "bullet" = 5, "laser" = 10, "energy" = 5, "bomb" = 50, "bio" = 100, "rad" = 95, "fire" = 100, "acid" = 90)
```
Comment (Ghommie) [inline code/modules/clothing/spacesuits/hardsuit.dm:224]: make it 100 red-proof at this point I dunno.

Comment (Trilbyspaceclone) [reply code/modules/clothing/spacesuits/hardsuit.dm:224]: I dont think it being rad proof is a good idea, if you have a CE suit its 105 and if you have a Engi suit is 100. Making it rad proof would kinda defeat that its just advance and not prefect

Comment (ghost) [reply code/modules/clothing/spacesuits/hardsuit.dm:224]: that was point of head only having 90

Comment (Trilbyspaceclone) [reply code/modules/clothing/spacesuits/hardsuit.dm:224]: Suits to cover head! This is were the famine is, all body parts /other/ then head are protected by the suit 100% if you ware a engi/CE suit
But the 10% off the head leads to a famine were you as a mob get 10% of rads regardless of your protection on body.

**Comment by lolman360** [comment]
the ce suit isnt radproof by design, you have to compromise between 100% radproofing (radsuit) and spaceproofing/tempproofing (cesuit).

powercreep.

**Comment by jakeramsay007** [comment]
That's not true in the slightest. You can already get full radiation immunity using the CE suit + CE jumpsuit @lolman360

**Comment by lolman360** [comment]
yes but only the ce jumpsuit so what's your point? there's like, 2 ce jumpsuits on station.

**Comment by Trilbyspaceclone** [comment]
@lolman360 CE suit dosnt protect head, meaning you CANT be rad proof with armor stacking

**Comment by lolman360** [comment]
yes that's LITERALLY what i just said with my last comment, the ce suit doesn't protect fully against rads

**Review by Putnam3145 — APPROVED** [review]
approving the code (it's fine) but not merging until people with a stronger opinion than me have a say and/or until i rework contamination

**Comment by dzahlus** [comment]
CE suit is almost exclusively used to meme the sm and fix a memed sm. I support any buff to its rad protection and it should just be 100% IMO. But then again only I am madlad enough to make an unironic triple co2 SM and get chernobyl'd even inside the suit

**Pull request merged**